eth-contracts/test/ethRewardsManager.test.js

```diff
@@ -30,11 +32,26 @@ contract('EthRewardsManager', async (accounts) => {
   let mockDelegateManager, mockStakingCaller, mockWormhole
 
   // intentionally not using acct0 to make sure no TX accidentally succeeds without specifying sender
-  const [, proxyAdminAddress, proxyDeployerAddress, staker, antiAbuseOracleAddress1, antiAbuseOracleAddress2, antiAbuseOracleAddress3] = accounts
+  const [
+    ,
+    proxyAdminAddress,
+    proxyDeployerAddress,
+    staker,
+    antiAbuseOracleAddress1,
+    antiAbuseOracleAddress2,
+    antiAbuseOracleAddress3
+  ] = accounts
   const tokenOwnerAddress = proxyDeployerAddress
   const guardianAddress = proxyDeployerAddress
-  const antiAbuseOracleAddresses = [antiAbuseOracleAddress1, antiAbuseOracleAddress2, antiAbuseOracleAddress3]
-  const recipient = Buffer.from('0000000000000000000000000000000000000000000000000000000000000000', 'hex')
+  const antiAbuseOracleAddresses = [
+    antiAbuseOracleAddress1,
+    antiAbuseOracleAddress2,
+    antiAbuseOracleAddress3
+  ]
+  const recipient = Buffer.from(
+    '0000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
The readability enthusiast in me says this could be a constant with a descriptive name. But probably only matters if we plan to use it in multiple places.

Reply:
Tests are horrible for readability right now. I didn't change this LOC but yeah... could take a better pass at readability on the whole. Maybe when we convert these tests to ts or write them in solidity itself

Reply:
this has some tips on writing tests with solidity https://remix-ide.readthedocs.io/en/latest/unittesting.html

my general impression though is hardhat / truffle like test suites are going to be a bit more advanced not to mention similar in to the tests in our other ts apps